enhancements/prevent-communitytasks-forbidden-events.md

```diff
@@ -0,0 +1,73 @@
+# Enhance community task fetch from external/public repo
+
+## Issue: [#SRVKP-1729](https://issues.redhat.com/browse/SRVKP-1729)
+
+If the firewall blocks requests to GitHub it will result in a lot of `forbidden` events logs
+
+
+## Proposal
+### Solution 1
+Add a toggle in the tektonconfig which a user can set to `false` if they don't want to install community tasks from repo
+
+- Implementation 1.1 
+    Add <span style="color: green"> enableCommunityTask: true </span> as spec element of addon
+
+```
+spec:
+  addon:
+    enablePipelinesAsCode: true
+    enableCommunityTask: true 
+    params:
+    - name: pipelineTemplates
+      value: "true"
+    - name: clusterTasks
+      value: "true"
+  config: {}
+  dashboard:
+    readonly: false
+```
+- Implementation 1.2
+    Add <span style="color: green"> enableCommunityTask: true </span> as Params of addon
+
+
+```
+spec:
+  addon:
+    enablePipelinesAsCode: true
+    params:
+    - name: pipelineTemplates
+      value: "true"
+    - name: clusterTasks
+      value: "true"
+    - name: enableCommunityTask  
+      value: "true"
+  config: {}
+  dashboard:
+    readonly: false
+```
+#### Pros
+- Easy implementation (less code change)
+- We don't even try to fetch the urls
+
+#### Cons
+- Needs a person to disable it manually setting the enableCommunityTask to false
+- Increase burden of the tektonconfig for one more toggle.
+- We don't even try to fetch the urls
+
+
+
+
+### Solution 2
```

Comment:
If we go that way, how can a user "force" a fetch later on (something changed and it's now open or something)